src/backend/gl/src/factory.rs

```diff
@@ -148,7 +148,7 @@ impl Factory {
         else {
             let usage = match info.usage {
                 Data => gl::STATIC_DRAW,
```

Comment:
I thought about it some more, let it be `STATIC_DRAW`, see the [discussion](https://gitter.im/gfx-rs/gfx?at=58724f1cc02c1a3959fedc57) in gitter.